dsc/docs-conceptual/dsc-2.0/tutorials/create-dsc-resource-machine-config.md

```diff
@@ -0,0 +1,2086 @@
+> specific to running the commands in PowerShell on a Windows computer.
+
+## Prerequisites
+
+- PowerShell or Windows PowerShell 5.1
+- VS Code with the PowerShell extension
+
+## 1 - Scaffold a DSC Resource module
+
+DSC Resources must be defined in a PowerShell module.
+
+### Create the module folder
+
+Create a new folder called `ExampleResources`. This folder is used as the root folder for the
+module and all code in this tutorial.
+
+```powershell
+New-Item -Path './ExampleResources' -ItemType Directory
+```
+
+```Output
+    Directory: C:\code\dsc
+
+Mode                 LastWriteTime         Length Name
+----                 -------------         ------ ----
+d----            9/8/2022 12:54 PM                ExampleResources
+```
+
+### Use VS Code to author the module
+
+Open the `ExampleResources` folder in VS Code. Open the integrated terminal in VS Code. Make sure
+your terminal is running PowerShell or Windows PowerShell.
+
+> [!IMPORTANT]
+> For the rest of this tutorial, run the specified commands in the integrated terminal at the root
+> of the module folder. This is the default working directory in VS Code.
+
+### Create the module files
+
+Create the module manifest with the `New-ModuleManifest` cmdlet. Use `./ExampleResources.psd1` as
+the **Path**. Specify **RootModule** as `ExampleResources.psm1` and **DscResourcesToExport** as
+`Tailspin`.
+
+```powershell
+$ModuleSettings = @{
+    RootModule           = 'ExampleResources.psm1'
+    DscResourcesToExport = 'Tailspin'
+}
+
+New-ModuleManifest -Path ./ExampleResources.psd1 @ModuleSettings
+Get-Module -ListAvailable -Name ./ExampleResources.psd1 | Format-List
+```
+
+```Output
+Name              : ExampleResources
+Path              : C:\code\dsc\ExampleResources\ExampleResources.psd1
+Description       :
+ModuleType        : Script
+Version           : 0.0.1
+PreRelease        :
+NestedModules     : {}
+ExportedFunctions :
+ExportedCmdlets   :
+ExportedVariables :
+ExportedAliases   :
+```
+
+Create the root module file as `ExampleResources.psm1`.
+
+```powershell
+New-Item -Path ./ExampleResources.psm1
+```
+
+```Output
+    Directory: C:\code\dsc\ExampleResources
+
+Mode                 LastWriteTime         Length Name
+----                 -------------         ------ ----
+-a---            9/8/2022  1:57 PM              0 ExampleResources.psm1
+```
+
+Create a script file called `Helpers.ps1`.
+
+```powershell
+New-Item -Path ./Helpers.ps1
+```
+
+```Output
+    Directory: C:\code\dsc\ExampleResources
+
+Mode                 LastWriteTime         Length Name
+----                 -------------         ------ ----
+-a---            9/8/2022  1:58 PM              0 Helpers.ps1
+```
+
+Open `Helpers.ps1` in VS Code. Add the following line, replacing `<Separator>` with `;` if you're on
+Windows and `:` if you're on Linux or macOS.
+
+```powershell
+$env:PSModulePath += "<Separator>$pwd"
```

Comment:
There is a dotnet approach that works crossplat. Thoughts?
```suggestion
$env:PSModulePath += "$([System.IO.Path]::PathSeparator)$pwd"
```

Reply:
That's a good fix!